docs/user-guide/configuring-at-tls-for-zowe-server.md

```diff
@@ -200,8 +204,8 @@ TTLSRule ApimlClientRule
   LocalAddr All
   LocalPortRange 1024-65535
   RemoteAddr All
-  RemotePortRange 7551-7555 # Range covers API ML services (gateway, zaas, discovery, api catalog, caching service)
-  Jobname ZWE1A* # Generate according to zowe.job.prefix in zowe.yaml
+  RemotePortRange 7552-7558 # Range covers API ML, app-server, and zss services
+  Jobname ZWE1AG # Generate according to zowe.job.prefix in zowe.yaml
```

Comment:
This should be generic to apply to all Zowe core services

Reply:
Do you mean you think the rule should be for all zowe servers to talk to each other?

Technically a rule like `jobname zwe1*` and `remoteportrange 7552-7558` does work, and covers more cases (like only zss + app-server running) but is also broader than necessary for apiml use (servers should only talk to gateway & discovery)?